packages/inertia/src/router.ts

```diff
@@ -152,11 +152,11 @@ export class Router {
   }
 
   protected isLocationVisitResponse(response: AxiosResponse): boolean {
-    return response && response.status === 409 && response.headers['x-inertia-location']
+    return !!(response && response.status === 409 && response.headers['x-inertia-location'])
```

Comment:
Curious — why did you make these two checks as part of this PR?

Reply:
I explained that in a previous comment, the return types where wrong because response had 'any' type and in v 0.24 of axios they removed the 'any' type so this problem appeared and Typescript would error